scripts/Makefile.connectal.build

```diff
@@ -231,11 +237,19 @@ ifeq ($(CONNECTAL_SHARED),1)
 	$(RUN_BSC_LM_LICENSE_FILE) $(GDB) ./bin/ubuntu.exe2 $(RUN_ARGS); retcode=$$?; exit $$retcode
 else
 ifeq ($(BOARD),bluesim)
+  ifeq ($(BSVSIM),)
     ifeq ($(RUNSOURCE2),)
 	$(RUN_BSC_LM_LICENSE_FILE) $(GDB) ./bin/ubuntu.exe $(RUN_ARGS); retcode=$$?; exit $$retcode
     else
 	$(RUN_BSC_LM_LICENSE_FILE) $(GDB2) ./bin/ubuntu.exe2& bsim2pid=$$!; $(GDB) ./bin/ubuntu.exe $(RUN_ARGS); retcode=$$?; kill $$bsim2pid; exit $$retcode
     endif
+  else
```

Comment:
Not being picky, but I don't see a difference between the code in "if" and "else" path?